CHANGELOG.md

```diff
@@ -0,0 +1 @@
+`functions:log` now supports v2 functions.
```

Comment:
These go out in public release notes. Make this vague or omit it from the relntoes entirely.

Reply:
removed